docs/modules/ROOT/pages/how-tos/testing/integration/group-snapshots.adoc

```diff
@@ -0,0 +1,75 @@
+= Creating a group snapshot
+
+In {ProductName}, the integration service provides `group` snapshot to allow users to submit multiple related pull/merge requests under one application to their repos, have them tested together and have the CI results posted back to pull/merge requests.
+
+== Procedures
+To have `group` snapshot created for multiple pull/merge requests, complete the following steps:
+
+. Identify the application and components that are related and need to be updated.
+. Navigate to their git repos.
+. Create branches with the same branch name for all of these git repos.
+. Commit code changes to branches created above and create pull/merge requests.
+
+After every build pipelinerun succeeds and component snapshot is created, group snapshot will be created for the new built component container images and other components. And then integration test runs for component and group snapshot.
+
+NOTE: Group snapshot is created only when there are at least two components with open pull/merge request and the same source branch name. 
+
+The created group snapshot is like below:
```

Comment:
Is it valid for users to create their own? If not, the level of detail in the CR might not be necessary.

Reply:
Users can't create their own group snapshot, the info in the annotation `test.appstudio.openshift.io/group-test-info` is requested by users, and then users can get code and run some test in their integration test task. This is why I paste the snapshot sample. Do you any good idea to show this annotation but not the whole snapshot info? Thanks!